tesh254/3f-api
# [Feature #160819695] All category endpoints.

#### What does this PR do?

It enables the creation, deletion, updating, getting any category name.

#### How should this be manually tested?

1. Launch PostMan, enter this URL  POST `{{url}}/api/v2/category` .
1. You must log in first to obtain a token.
1. Copy and paste the token into the `Authorization` header.
1. Send the request to the server with a json body as shown
```json 
{
    "category_name":"Fried"
}
```
* If successful the server's response will be:
1. `201 Created` 
1. Order details including total amount to be paid
1. If the order is successful.

* If a bad request:
1. `400 Bad Request` if a field is missing.
1. You haven't provided a token.

* To get a category and see the contents we use and an identifier for going through the DB to obtain it, if not found a `404 Not Found` will be the server response

#### Background Context
Please use python3 versions ranging from `3.5`, `3.6`, 3.7`.

#### Pivotal Tracker Stories
For this project management stories, you can get them at [https://www.pivotaltracker.com/projects/2196342](https://www.pivotaltracker.com/projects/2196342) 
The stories are public anyone can see them update any if need be.    

[#160819695](https://www.pivotaltracker.com/story/show/160819695)
